src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -220,43 +221,116 @@ osx_output_set_device(OSXOutput *oo, Error &error)
 	return ret;
 }
 
+
+/*
+	This function (the 'render callback' osx_render) is called by the
```

Comment:
@MaxKellermann is this an OK style for multi-line comments? If not, could you point me to a file in the tree that shows the right style for such comments?